docs/corsair-commander-core-guide.md

```diff
@@ -0,0 +1,42 @@
+# Corsair Commander Core
+_Driver API and source code available in [`liquidctl.driver.commander_core`](../liquidctl/driver/commander_core.py)._
+
+Currently, functionality implemented is listed here. More is planned to be added.
+
+## Initializing the device
+
+The device should be initialized every time it is powered on.
+
+```
+# liquidctl initialize
+Corsair Commander Core (experimental)
+├── Firmware version            1.6.135  
+├── AIO LED count                    29  
+├── RGB port 1 LED count              8  
+├── RGB port 2 LED count              8  
+├── RGB port 3 LED count            N/A  
+├── RGB port 4 LED count            N/A  
+├── RGB port 5 LED count            N/A  
+├── RGB port 6 LED count            N/A  
+├── Water temperature sensor        Yes  
+└── Temperature sensor 1             No   
+```
+
+## Retrieving the pump speed, fan speeds, and temperatures
+
+
+The commander core currently can retrieve the pump speed, fan speeds, temperature of the water, and
```

Comment:
The last commit made `Commander Core` all-lowercase (probably an accident).

Reply:
This still was not fixed

Reply:
e34c155b535e20e88cab23dd7b7f91238ceedb2f